libraries/AP_InertialSensor/AP_InertialSensor_Invensensev2.cpp

```diff
@@ -639,9 +640,8 @@ bool AP_InertialSensor_Invensensev2::_hardware_init(void)
         return false;
     }
 
-    // setup for register checking. We check much less often on I2C
-    // where the cost of the checks is higher
-    _dev->setup_checked_registers(7, _dev->bus_type() == AP_HAL::Device::BUS_TYPE_I2C?200:20);
+    // disabled setup of checked registers as it can't cope with bank switching
+    // _dev->setup_checked_registers(7, _dev->bus_type() == AP_HAL::Device::BUS_TYPE_I2C?200:20);
```

Comment:
The fix already  merged, didnt it?

Reply:
@sh83 this PR was based of yesterday's master, so do you have a PR to reference where it was previously merged?

Reply:
it wasn't merged yet, but it is now